src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,69 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidUserDataException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int LOGIN_MIN_LENGTH = 3;
+    private static final int LOGIN_MAX_LENGTH = 20;
+    private static final int PASSWORD_MIN_LENGTH = 6;
+    private static final int PASSWORD_MAX_LENGTH = 25;
+    private static final int MIN_AGE = 18;
+    private static final int MAX_AGE = 100;
```

Comment:
let's remove this constant and check, because we have people with age greater than 100)